src/main/groovy/org/labkey/gradle/plugin/ServerDeploy.groovy

```diff
@@ -420,102 +359,6 @@ class ServerDeploy implements Plugin<Project>
         }
         project.tasks.named('deployApp').configure {dependsOn(project.tasks.named("checkModuleTasks"))}
         project.tasks.named('checkModuleTasks').configure {mustRunAfter(project.tasks.stageApp)} // do this so the message appears at the bottom of the output
-        if (project.plugins.hasPlugin(Tomcat)) {
-            if (project.tomcat.hasCatalinaHome()) {
-                project.tasks.named("cleanBuild").configure {
-                    it.dependsOn(project.tasks.stopTomcat)
-                }
-                project.tasks.named("cleanDeploy").configure {
-                    it.dependsOn(project.tasks.stopTomcat)
-                }
```

Comment:
Have you tried `cleanBuild` and/or `cleanDeploy` without this dependency on `stopTomcat`. I'm guessing we still want the task dependencies, just don't need the if statements around them.

Reply:
I restored the two `dependsOn(stopTomcat)` calls (now unconditional)